docs/RULES.md

```diff
@@ -266,7 +266,7 @@ lines inside code blocks.
 
 Tags: line_length
 
-Parameters: line_length (number; default 80)
+Parameters: line_length, code_blocks (number; default 80, boolean; default true)
```

Comment:
Found no examples of non-uniform parameters, this is the best style I came up with that also passes tests.